### ISSUE TITLE
NVDA reads extraneous information in Task Manager

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#13250]
### Is your feature request related to a problem? Please describe.

1. Open the task manager in Windows 10 (control+shift+escape)
2. Navigate through the objects (Either in the original "Fewer details" view, or the main processes view of "More details")

NVDA reads:

List of items  data grid
List of items  list
Process: Google Chrome

then when you press down arrow:

List of items  data grid
List of items  list
Process: Microsoft Word

and again:

List of items  data grid
List of items  list
Process: Firefox

(the process name changes, but the other information is read before every item.)

### Describe the solution you'd like

NVDA should read the list type (is it a data grid or a list?) only once, the first time the focus moves to this section.  Each time down arrow is pressed, only the process should be reported.

Ideally, the name should be read first and then process.

(On opening the window or tabbing to the list):

List of items  list (I assume it is a list called "List of items" or something similar?)
Google Chrome, process.

then on down arrow

Microsoft Word, process
Firefox, process

etc

Ideally the process name should be read before identifying it as a process.  Otherwise, as you down arrow you hear "process, process, process".

### Describe alternatives you've considered

Currently, the quickest way to have the process name read is to:
1) Press down arrow to move to the next process
2) Quickly press numpad 8 / NVDA+shift+. to read the current line in review.

### Additional context

Tested using NVDA 2021.3.1 
on Windows 10 (64-bit) Version: 21H2 (2009), Build: 19044.1466

**Comment by enessaribas** [comment]
hi,
I cannot reproduce this on Windows 11.

**Comment by Qchristensen** [comment]
I just checked with the latest alpha-24481,90922270 - still on Windows 10 (64-bit) Version: 21H2 (2009), Build: 19044.1466  - and it behaves the same as with 2021.3.1 did for me.  Looks like the behaviour might be different in Windows 11

**Comment by mazen428** [comment]
Hello.
I think  task manager has some sort of MSAA and UIA proxy working at the same time, looking at the UIA provider of the list of items list, it says.
"UIA providerDescription: [pid:10688,providerId:0x0 Main(parent link):Microsoft: MSAA Proxy (unmanaged:UIAutomationCore.dll)]".
I think that might be why it's saying list of items data grid, then list of items list, because it's getting double focus events from something else.
I'm not sure how to filter those out, still new to NVDA's code.

**Comment by cary-rowen** [comment]
hello @enessaribas 
To reproduce the issue you should check the "Enable selective registration for UI Automation events and property changes" option in the NVDA advanced settings panel.
This option actually requires a restart of NVDA to take effect.
So we can't solve this annoying problem by simply creating a separate configuration file for the task manager.
Thanks

**Comment by Adriani90** [comment]
cc: @michaelDCurran, @leonardder

**Comment by seanbudd** [comment]
I believe this is a duplicate of #11599, closing

**Comment by sativarg-tuta** [comment]
Hello NVDA community,
When I start Windows Task Manager and in the Processes Tab I get the redundant information I restart NVDA with alt control N, and then use alt tab to get back to task manager and the behavior of NVDA is as expected... no more annoying repeated location 
information until I open the manager again. 
The identifier, "process" is read then the name.
When a category is reached like background processes, windows applications that is read only once.
Usually when I use Program Manager I am needing information quickly on what is happening and having to restart NVDA means precious seconds lost?
If I knew how to make a NVDA profile for Task Manager and the right things to disable I could probably do away with a lot of unnecessary delays... but I am just not that bright and or too lazy to learn.
I could make a script for starting Task manager that would start it then restart NVDA and send NVDA focus into the list... but as I said I am not that bright these days...
thanks for the great resource and work community...
blessings
chuck 🔥💖🔥
